README.md

```diff
@@ -71,11 +71,9 @@ Use the built-in query workbench to prototype [DruidSQL](https://druid.apache.or
 
 ### Documentation
 
-You can find the [documentation for the latest Druid release](https://druid.apache.org/docs/latest/) on
-the [project website](https://druid.apache.org).
+The [latest documentation](https://druid.apache.org/docs/latest/) is published with each official release to the [project website](https://druid.apache.org), where documentation for [previous releases](https://druid.apache.org/docs/) is also made available.
 
-If you would like to contribute documentation, please do so under
-`/docs` in this repository and submit a pull request.
+Make documentation and tutorials updates in [`/docs`](https://github.com/apache/druid/tree/master/docs) using [MarkDown](https://www.markdownguide.org/) and contribute them using a Pull Request.
```

Comment:
I think pull request is not a proper now: "pull request"
Also, since we don't currently have a guide for docs contributors, maybe we can say that they should read the Contributing section of the Community guide https://druid.apache.org/community/

Reply:
Agreed, the usual term is "pull request" or "PR".

I did a doc change way back when. I seem to recall there are instructions for how to set up the build system/checker/whatever we use. I seem to recall it will build the docs, complain about your boo-boos, and serve the results as a web page. We should link to that info (assuming I'm not imagining it) here.

Reply:
Done :)